vendor/paella_overrides/src/01_utils.js

```diff
@@ -0,0 +1,646 @@
+		if (params) {
+			className = params.className;
+			closeButton = params.closeButton;
+			width = params.width;
+			height = params.height;
+			onClose = params.onClose;
+		}
+
+		this.doShowElement(domElement,closeButton,width,height,className,onClose);
+	},
+
+	showMessage:function(message,params) {
+		var closeButton = true;
+		var width = "60%";
+		var height = "40%";
+		var onClose = null;
+		var className = this.messageClassName;
+		if (params) {
+			className = params.className;
+			closeButton = params.closeButton;
+			width = params.width;
+			height = params.height;
+			onClose = params.onClose;
+		}
+
+		this.doShowMessage(message,closeButton,width,height,className,onClose);
+	},
+
+	doShowElement:function(domElement,closeButton,width,height,className,onClose) {
+		this.onClose = onClose;
+		$('#playerContainer').addClass("modalVisible");
+
+		if (this.currentMessageBox) {
+			this.close();
+		}
+		if (!className) className = this.messageClassName;
+
+		if (!width) { width = '80%'; }
+
+		if (!height) { height = '30%'; }
+
+		var modalContainer = document.createElement('div');
+		modalContainer.className = this.modalContainerClassName;
+		modalContainer.style.position = 'fixed';
+		modalContainer.style.top = '0px';
+		modalContainer.style.left = '0px';
+		modalContainer.style.right = '0px';
+		modalContainer.style.bottom = '0px';
+		modalContainer.style.zIndex = 999999;
+
+		var messageContainer = document.createElement('div');
+		messageContainer.className = className;
+		messageContainer.style.width = width;
+		messageContainer.style.height = height;
+		messageContainer.style.position = 'relative';
+		messageContainer.appendChild(domElement);
+		modalContainer.appendChild(messageContainer);
+
+		$('body')[0].appendChild(modalContainer);
+
+		this.currentMessageBox = modalContainer;
+		this.messageContainer = messageContainer;
+		var thisClass = this;
+		this.adjustTop();
+
+		if (closeButton) {
+			this.createCloseButton();
+		}
+	},
+
+	doShowMessage:function(message,closeButton,width,height,className,onClose) {
+		this.onClose = onClose;
+		$('#playerContainer').addClass("modalVisible");
+
+		if (this.currentMessageBox) {
+			this.close();
+		}
+		if (!className) className = this.messageClassName;
+
+		if (!width) { width = '80%'; }
+
+		if (!height) { height = '30%'; }
+
+		var modalContainer = document.createElement('div');
+		modalContainer.className = this.modalContainerClassName;
+		modalContainer.style.position = 'fixed';
+		modalContainer.style.top = '0px';
+		modalContainer.style.left = '0px';
+		modalContainer.style.right = '0px';
+		modalContainer.style.bottom = '0px';
+		modalContainer.style.zIndex = 999999;
+
+		var messageContainer = document.createElement('div');
+		messageContainer.className = className;
+		messageContainer.style.width = width;
+                // #DCE MATT-2429 fix Flash message height
+		messageContainer.style.minHeight = height;
+		messageContainer.style.position = 'relative';
```

Comment:
I think we actually need a fixed minimum height AND a height. The height value would be 30% and the minHeight would be the minimum number of pixels to fit our message (I believe we figured out that this was 170px)

I would swap this out for:

```js
messageContainer.style.height = height;
messageContainer.style.minHeight = '148px';
```

I hope I'm understanding all this correctly!

Reply:
@gabeabrams the combination of style on the player at when showing the errorDisplay, is such that the width is flexible and the display box adjusts to the size of the window. This is useful because users can see the message at any window width. Giving the error display a fixed height prevents the display from containing it's contents when the browser window is squished vertically, such as in portrait mode. The contents spill out beyond the fixed container's height. Which is good in that the content is still visible, but bad because it looks terrible. Using style.height empty, the error container is behaving infinitely flexible (vertically) and contains the content at all window widths. This is the (semi-old text) with minHeight, but no height set on the displaybox. It is taller than 170px.  https://files.slack.com/files-pri/T02T5MHL9-FD81TVAJW/image.png